prospector-more.yml

```diff
@@ -0,0 +1,47 @@
+inherits: prospector
+
+strictness: medium
+
+ignore-paths:
```

Comment:
Should put a comment here or something inviting folks to remove a directory and fix the errors going forward, perhaps.